src/improvements/tasks/sep/005-deputy-pause-key-rotation/README.md

```diff
@@ -0,0 +1,32 @@
+# 005-deputy-pause-key-rotation Deputy Pause Module Rotation key
+
+Status: [DRAFT]()
+
+## Objective
+
+Rotates the `DeputyPauseModule` Deputy EOA from `0xfcb2575ab431a175669ae5007364193b2d298dfe` to `0x6A07d585eddBa8F9A4E17587F4Ea5378De1c3bAc`. 
+To achieve this, we will need to use the `setDeputy` function of the `DeputyPauseModule` contract deployed at `0xc6f7C07047ba37116A3FdC444Afb5018f6Df5758`.
+
+![CleanShot 2025-04-10 at 10 15 35@2x](https://github.com/user-attachments/assets/042bbb15-a19b-4edf-bff1-e79adaf4e2ce)
+
+### Execution Timing
+
+Expected to be executed the **10 APRIL 2025**.
+
+## Simulation
+
+Please see the "Simulating and Verifying the Transaction" instructions in [SINGLE.md](../../../SINGLE.md).
+When simulating, ensure the logs say `Using script /your/path/to/superchain-ops/src/improvements/template/DeputyPauseRotationKey.sol`.
```

Comment:
nit, path should be updated with this folder

Reply:
wdym here?

Reply:
oops sorry I misread, you can ignore this